graphics/poppler/Portfile

```diff
@@ -10,6 +10,7 @@ PortGroup           legacysupport 1.0
 name                poppler
 conflicts           xpdf-tools
 version             0.83.0
+revision            1
```

Comment:
There is no reason to increment the `revision` because users must explicitly install a new variant. So set this back to 0.